src/Symfony/Component/HttpClient/Response/TraceableResponse.php

```diff
@@ -52,24 +54,36 @@ public function getHeaders(bool $throw = true): array
 
     public function getContent(bool $throw = true): string
     {
-        $this->content = $this->response->getContent(false);
-
         if ($throw) {
-            $this->checkStatusCode($this->response->getStatusCode());
+            try {
+                $this->checkStatusCode($this->response->getStatusCode());
+            } catch (HttpExceptionInterface $httpException) {
+                try {
+                    return $this->content = $this->response->getContent(false);
+                } catch (TransportExceptionInterface $e) {
+                    throw $httpException ?? $e;
```

Comment:
`$httpException` is never null in this block

Reply:
fixed, thanks